vm_supervisor/utils.py

```diff
@@ -52,3 +53,23 @@ async def run_and_log_exception(coro: Coroutine):
 def create_task_log_exceptions(coro: Coroutine, *, name=None):
     """Ensure that exceptions running in coroutines are logged."""
     return asyncio.create_task(run_and_log_exception(coro), name=name)
+
+
+async def run_in_subprocess(
+    command: List[str], check: bool = True, stdin_input: Optional[bytes] = None
```

Comment:
The encoding parameter is needed for some commands related with devmapper

Reply:
Stdin and stdout are always expected to be bytes. Passing an extra parameter to ask to `encode()` / `decode()` would not make the code clearer or easier to manipulate.